test/CMakeLists.txt

```diff
@@ -212,6 +213,9 @@ add_rev_test(ZICBOM zicbom 45 "all;memh;rv64" SCRIPT "run_zicbom.sh")
 add_rev_test(BACKINGSTORE backingstore 100 "all;rv64" SCRIPT "run_backingstore.sh")
 add_rev_test(MEM_DUMP mem_dump 10 "all;rv64" SCRIPT "run_mem_dump.sh")
 add_rev_test(LWSP lwsp 30 "all;memh;rv64")
+add_rev_test(RDCYCLE rdcycle 5 "all;rv64")
+add_rev_test(RDTIME rdtime 5 "all;rv64")
+add_rev_test(RDINSTRET rdinstret 5 "all;memh;rv64")
```

Comment:
`rdcycle` and `rdtime` differences measure an indeterminately long time to execute a fixed number of instructions because of stalls, so MemH is disabled for them. But `rdinstret` counts retired instructions, not the time it took, so it works for both RevMem and MemH.